### ISSUE TITLE
Reports the error ' Can't perform a React state update on an unmounted component'

### ISSUE THREAD

**Issue by chris-peerman** [simoneb/axios-hooks#74]
```
index.js:1375 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    in TestComponent (at App.js:23)
Uncaught (in promise) Cancel {message: undefined}
```

Example here [test-axios-hook.zip](https://github.com/simoneb/axios-hooks/files/3824047/test-axios-hook.zip), to use the example:
1. Run the example
2. In Chrome devtools click Network, and set the Throttling to 'Slow 3G'
3. Click 'Reload' and before it completes click 'Hide'

You'll then get the following error:
![image](https://user-images.githubusercontent.com/1798567/68468523-1b5c7d80-0210-11ea-8a05-db514c75864f.png)

**Comment by chris-peerman** [comment]
This relates to the recent pull request https://github.com/simoneb/axios-hooks/pull/67 which was aimed at fixing this.

**Comment by simoneb** [comment]
@chris-peerman thanks for the repro. can you check if the latest prerelease version 1.7.2-0 fixes the issue?

**Comment by chris-peerman** [comment]
Thanks @simoneb I've tried it using the test harness. I'll integrate it soon into our app and make sure it is working there too.

**Comment by simoneb** [comment]
Fixed in 1.7.2
